src/components/SiteHeader/SiteHeader.tsx

```diff
@@ -24,39 +26,43 @@ const Headers: Header[] = [
   { href: '/about', title: 'about' },
 ]
 
-const SiteHeader = (props: Props) => {
+const SiteHeader = (props: { location: GatsbyLocation }) => {
   const pathname: string = _.get(props, 'location.pathname', '/')
   const isRoot: boolean = _.eq(pathname, '/')
+
+  const theme: HeaderTheme = _.startsWith(pathname, '/photography')
+    ? HeaderTheme.LIGHT
+    : HeaderTheme.DARK
+
   return (
-    <Headroom
-      className="bg-gray-primary"
-      style={{
-        MozTransition: 'all 0.7s ease-in-out',
-        OTransition: 'all 0.7s ease-in-out',
-        WebkitTransition: 'all 0.7s ease-in-out',
-        transition: 'all 0.7s ease-in-out',
-      }}
-    >
-      <header
+    <Headroom>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)